src/core/raster/qgscolorrampshader.h

```diff
@@ -72,6 +72,11 @@ class CORE_EXPORT QgsColorRampShader : public QgsRasterShaderFunction
     /** \brief Get the color ramp type */
     QgsColorRampShader::ColorRamp_TYPE colorRampType() const {return mColorRampType;}
 
+    /** \brief Get the original color ramp name
+     * @note added in QGIS 3.0
+     */
```

Comment:
It's nice to link these getters and setters with doxygen `@see` tags. Ie

```
/** \brief Get the original color ramp name
  * @note added in QGIS 3.0
  * @see setColorRampName()
  */
```

(and also link the setter back to the getter).